gnosis/safe-react
# Refactor: remove unnecessary action, refactor safe removal

## What it solves
I was browsing the code and found this.

My pr removes unnecessary action that didn't do anything besides calling the correct one. Safes are persisted locally in the middleware (src/logic/safe/store/middleware/safeStorage.ts) It's not needed to call storage functions directly

## How this PR fixes it
By using the correct action
Also, move safe reducer types to `logic/safe/store` folder 

## How to test it
1. Remove a safe, remove a default safe
2. Should work fine and the safe shouldn't be there after reloaded
